src/app/components/Account.tsx

```diff
@@ -1,30 +1,56 @@
 import 'index.css'
+import * as React from 'react'
+import classnames from 'classnames'
+import { useNavigate } from 'react-router-dom'
 
 import { useAuth } from 'app/hooks/useAuth'
-
 import SvgAccount from './SVG/SvgAccount'
 
 export default function Account() {
   const { user } = useAuth()
+  const navigate = useNavigate()
+  const [popup, setPopup] = React.useState(false)
+  const popupStyle = classnames(styles.popup, { hidden: !popup })
 
   const avatar =
     'https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/Flag_of_Ukraine.svg/375px-Flag_of_Ukraine.svg.png'
 
+  const handleOnCLickAccount = () => {
+    setPopup(!popup)
+  }
+  const handleOnCLickLogout = () => {
+    localStorage.removeItem('jwtAccessToken')
+    localStorage.removeItem('jwtRefreshToken')
+    navigate('/login')
+  }
+
   if (!user) {
     return <></>
   }
 
   return (
-    <div className={styles.container}>
-      <img className={styles.container__image} alt="avatar" src={avatar} />
-      <div className={styles.container__rightblock}>
-        <div className={styles.rightblock__text}>
-          <span className={styles.text__name}>{user.username}</span>
-          <span className={styles.text__nickname}>{'@' + user.nickname}</span>
+    <>
+      <div className={popupStyle}>
+        <div className={styles.popup__logout} onClick={handleOnCLickLogout}>
+          Log out @{user.nickname}
         </div>
-        <SvgAccount />
+        <svg className={styles.popup__triangle} viewBox="0 0 24 24">
+          <g>
+            <path d="M22 17H2L12 6l10 11z" />
+          </g>
+        </svg>
```

Comment:
Move it to separate component. You may even store it in this file if it specific for this component